pkg/apis/core/v1beta1/helper/managedseed_test.go

```diff
@@ -0,0 +1,156 @@
+// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package helper_test
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/ptr"
+
+	gardencorev1beta1 "github.com/gardener/gardener/pkg/apis/core/v1beta1"
+	v1beta1constants "github.com/gardener/gardener/pkg/apis/core/v1beta1/constants"
+	. "github.com/gardener/gardener/pkg/apis/core/v1beta1/helper"
+)
+
+var _ = Describe("#ReadManagedSeedAPIServer", func() {
```

Comment:
For consistency with the other tests structure:
```suggestion
var _ = Describe("Helper", func() {

	Describe("#ReadManagedSeedAPIServer", func() {
```